lib/python/packagekit/backend.py

```
@@ -470,6 +470,13 @@ def remove_packages(self, transaction_flags, package_ids, allowdep, autoremove):
         '''
         self.error(ERROR_NOT_SUPPORTED, "This function is not implemented in this backend", exit=False)
 
+    def purge_packages(self, transaction_flags, package_ids, allowdep, autoremove):
```

Ruff (0.15.15)

[warning] Unused method argument: `transaction_flags`

(ARG002)